src/coreclr/src/tools/ILVerification/Strings.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!-- 
-    Microsoft ResX Schema 
-    
+  <!--
```

Comment:
These whitespaces are added by Visual Studio resx file editor. They will be back next time somebody edits this file in VS. Could you please undo the whitespace changes in this file?

Reply:
I have double checked on Windows 10; resx editor in VS 2019 does not add extra spaces on modifying the existing resx file.